.circleci/config.yml

```diff
@@ -211,6 +211,19 @@ jobs:
 
     steps: *std_build_steps
 
+  build_bullseye_64bit_debs:
```

Comment:
also need to include it down by `test_and_ship`

Reply:
Included, thanks.